workflows/templates/workflow-template.yaml

```diff
@@ -0,0 +1,141 @@
+                - name: IMAGE_STREAM_REGISTRY
+                  value: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}"
+                - name: IMAGE_STREAM_NAMESPACE
+                  value: "{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}"
+                - name: IMAGE_STREAM_TAG
+                  value: "{{inputs.parameters.IMAGE_STREAM_TAG}}"
+
+        - - name: process-information
+            template: process
+            arguments:
+              parameters:
+                - name: REPOSITORY
+                  value: "{{inputs.parameters.REPOSITORY}}"
+                - name: IMAGE_STREAM_REGISTRY
+                  value: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}"
+                - name: IMAGE_STREAM_NAMESPACE
+                  value: "{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}"
+                - name: IMAGE_STREAM_TAG
+                  value: "{{inputs.parameters.IMAGE_STREAM_TAG}}"
+
+    - name: analyse
+      inputs:
+        parameters:
+          - name: REPOSITORY
+          - name: IMAGE_STREAM_REGISTRY
+          - name: IMAGE_STREAM_NAMESPACE
+          - name: IMAGE_STREAM_TAG
+      container:
+        image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}
+                /{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}
+                /srcopsmetrics:{{inputs.parameters.IMAGE_STREAM_TAG}}"
+        command: ["srcopsmetrics/cli.py"]
+        args: ["-c", "{{inputs.parameters.REPOSITORY}}"]
+        env:
+          - name: GITHUB_ACCESS_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: sesheta-srcops
+                key: github-access-token
+          - name: S3_ENDPOINT_URL
+            valueFrom:
+              configMapKeyRef:
+                name: thoth
+                key: ceph-host
+          - name: CEPH_BUCKET
+            valueFrom:
+              configMapKeyRef:
+                name: thoth
+                key: ceph-bucket
+          - name: CEPH_BUCKET_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                name: thoth
+                key: ceph-prefix
+          - name: CEPH_SECRET_KEY
+            valueFrom:
+              secretKeyRef:
+                name: thoth
+                key: ceph-secret-key
+          - name: CEPH_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                name: thoth
+                key: ceph-key-id
+          - name: PYTHONPATH
+            value: "."
+
+    - name: process
+      inputs:
+        parameters:
+          - name: REPOSITORY
+          - name: IMAGE_STREAM_REGISTRY
+          - name: IMAGE_STREAM_NAMESPACE
+          - name: IMAGE_STREAM_TAG
+      container:
+        image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}
+                /{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}
+                /srcopsmetrics:{{inputs.parameters.IMAGE_STREAM_TAG}}"
+        command: ["srcopsmetrics/cli.py"]
+        args: ["-p", "{{inputs.parameters.REPOSITORY}}"]
```

Comment:
We should avoid talking to Ceph and let Argo do the work, we should store locally and reuse the local storage, therefore you need to add a volume in the main workflow that can be reused across steps. Have a look at: https://github.com/thoth-station/adviser/blob/313889b699532169d7bd5fab1be3cf1d7b903fad/openshift/adviser-template.yaml#L160 for the volume set and to how they are stored locally for the other steps and on Ceph by the workflow https://github.com/thoth-station/adviser/blob/313889b699532169d7bd5fab1be3cf1d7b903fad/workflows/templates/advise-template.yaml#L42